setup.py

```diff
@@ -6,6 +6,7 @@
 src_dir = os.path.dirname(__file__)
 
 install_requires = [
+    "awscli>=1.11.37",
```

Comment:
This is a pretty heavy dependency for what amounts to about [80 LOC](https://github.com/aws/aws-cli/blob/10cbd8d6aec62a0e02e3e75d9624559a703fca4f/awscli/customizations/cloudformation/yamlhelper.py).

I think I'd rather just vendor that in stacker, rather than adding a dependency on the AWS CLI.